runtime/src/lib.rs

```diff
@@ -614,7 +633,7 @@ impl pallet_aggregate::Config for Runtime {
     type RuntimeHoldReason = RuntimeHoldReason;
     type AggregationSize = AggregateMaxSize;
     type MaxPendingPublishQueueSize = AggregateQueueSize;
-    type ManagerOrigin = EnsureRoot<AccountId>;
+    type ManagerOrigin = EitherOfDiverse<EnsureRoot<AccountId>, Treasurer>;
```

Comment:
As of now, perhaps, it doesn't make much sense to have `Treasurer` here...
@la10736 what do you think ?

Reply:
I don't think `Treasurer` is correct here.: the manager here is a just a technical maintainer that eventually can solve issues. Also in a future where we'll remove `sudo` and use governace for runtime upgrade I guess that here we'll have a multiginature account instead.

Reply:
Alright, reverted the change